docs/.vitepress/vitepress/components/theme-editor/ep-theme-drawer.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import { Brush } from '@element-plus/icons-vue'
+import { useThemeStore } from '~/store/theme'
+
+const drawer = ref(import.meta.env.DEV)
+const direction = ref('rtl')
+
+const tStore = useThemeStore()
+
+const downloadTheme = () => {
+  // eslint-disable-next-line no-alert
+  alert('TODO')
+}
+
+const uploadTheme = () => {
+  // eslint-disable-next-line no-alert
+  alert('TODO')
+}
+</script>
+
+<template>
+  <div>
+    <el-drawer v-model="drawer" :lock-scroll="false" :direction="direction">
+      <template #header>
+        <span class="flex-1" text="sm">Theme Editor</span>
+      </template>
+      <div class="-mt-4">
+        <EpThemePrimaryColors />
+        <EpThemePrimary />
+        <EpThemeSecondaryColors />
+
+        <el-button class="flex" w="full" @click="tStore.reset()">
+          <i-ep-refresh />
+          全部恢复默认配置
```

Comment:
i18n

Reply:
Now the manual method of I18N is not a suitable solution, I plan to introduce vue-i18n in the new PR.